### DISCUSSION TITLE
How to disable viewing SimplyHTML window?

### DISCUSSION THREAD

**Discussion by OpaYoda40** [freeplane/freeplane#1027] [Help]
-- This perennial issue since early start out of FreePlane fledging from FreeMIND.  Came up while using v 1.9.12 (OSX-intel).  Popped up while using keyboard shortcut to visually link two nearby nodes in same subtree.  
  -- Researched best I could only to find deprecated comment in Freeplane fora & non-functional advice in general tech community fora. 
  --  Had hoped to resolve by downloading/installing v 1.10,6 (OSX-intel), but no joy.  Once mounted, the mount comes up early, before any windowing modules are loaded.  shutting down & rebooting FP does not kill it. The mount point must be deep & early in the boot up code.  
  --  That would not be much of a problem if there were simple on/off switch options.  So far I have found ANY pull down menu or keyboard shortcut options to do so,  Preferred my case would to have graphical on/off option in the window it self.  This, fo eliminate need to switch hands/fingers between mouse & keyboard.  (Obviously keyboard shortcuts needed for visually challenged.)

**Comment by EdoFro** [comment]
`View->Notes->Display note panel (Ctrl+Shift+Less)`

`Preferences…->Behaviour->In-line node editor->Display inline editor for all new nodes` set to **false** (unchecked)
